utils/checks.stanza

```diff
@@ -408,23 +412,31 @@ public defn check-single-push-pull-levels (out-p:JITXObject, in-p:JITXObject) :
       subcheck-description = "Verify input pin is tri-stateable to be compatible with output pin type",
       pass-message =         "Input pin %_ is tri-stateable and is compatible with the output pin %_" % [ref(in-p), ref(out-p)],
       fail-message =         "Input pin %_ is not tri-stateable and is not compatible with the output pin %_" % [ref(in-p), ref(out-p)],
-      locators =             [in-p]
     )
+      ; locators =             [in-p]
 
 public pcb-check connected (p:JITXObject|Tuple<JITXObject>) :
   val name = "connected net checks"
   val description = "Check that the object(s) are connected"
   val category = "net-check"
+  val pass-message =
+    match(p) :
+      (p:JITXObject) : "Object %_ is connected" % [ref(p)]
+      (p:Tuple<JITXObject>) : "Objects %, are connected" % [map(ref, p)]
+  val fail-message =
+    match(p) :
+      (p:JITXObject) : "Object %_ is not connected" % [ref(p)]
+      (p:Tuple<JITXObject>) : "Objects %, are not connected" % [map(ref, p)]
```

Comment:
We probably want the messages to change depending on what's passed in!
We could also branch based on the number of elements in the tuple. Zero, one, or multiple could be handled differently.

Reply:
Good idea!